src/Store/ExtensionClient.re

```diff
@@ -330,7 +330,7 @@ let create = (~extensions, ~setup: Setup.t) => {
   };
 
   let onShowMessage = message => {
-    dispatch(Actions.ShowNotification(Notification.create(message)));
+    dispatch(Actions.ExtMessageReceived(message));
```

Comment:
Because notifications are now created through an effect rather than directly though a "function-call-action", we have to dispatch a more specific event-like action and handle that to emit an effect. Which is how it should have been done anyway, so a nice side-effect.

Reply:
Thanks for fixing this!